lesson_06/homeworks_06.py

```diff
@@ -1,72 +1,73 @@
-"""
+if alien_color == 'green':
+    print("Гравець щойно заробив 5 балів.")
 
-# task 3
-# task 4
-"""  Скопіюйте пеопередню відповідь, змініть і доповніть її умовою elif.
-змінну під назвою alien_color перетворіть у список значень: 'green', 'yellow', 'red'
-Якщо колір прибульця зелений, надрукуйте, що гравець щойно заробив 5 балів.
-Якщо колір прибульця не зелений, надрукуйте, що гравець щойно заробив 10 балів.
-Якщо прибулець червоний, надрукуйте повідомлення про те, що гравець заробив 15 очок
-+ напишіть цикл for що перебере і обробить всі значення списку alien_color
-"""
+alien_color = 'yellow'
 
-# task 5
-"""  Начинки для піци (pizza_topping): напишіть цикл, який пропонує користувачеві ввести ряд начинок
-для піци, доки він не введе значення 'quit'. Коли вони введуть кожну начинку,
-надрукуйте повідомлення про те, що ви додасте цю начинку до їхньої піци.
-"""
+if alien_color == 'green':
+    print("Гравець щойно заробив 5 балів.")
+else:
+    print("Гравець щойно заробив 10 балів.")
+
+alien_colors = ['green', 'yellow', 'red']
+
+for alien_color in alien_colors:
+    if alien_color == 'green':
+        print("Гравець щойно заробив 5 балів.")
+    elif alien_color == 'red':
+        print("Гравець заробив 15 очок.")
+    else:
+        print("Гравець щойно заробив 10 балів.")
+
+while True:
+    topping = input("Введіть начинку для піци (або 'quit' для завершення): ")
+    if topping == 'quit':
+        break
+    print(f"Додаємо {topping} до вашої піци.")
 
 # task 6
-"""  Напишіть програму, яка знаходить суму всіх цифр натурального числа, яке вводить користувач.
-Для перебору вводу від користувача використовуйте цикл. Виведіть суму цифр числа на екран.
-Приклад виконання програми:
-Введіть натуральне число: 12345
-Сума цифр числа 12345: 15
-"""
+number = input("Введіть натуральне число: ")
+sum_of_digits = 0
+for digit in number:
+     sum_of_digits += int(digit)
+     rint(f"Сума цифр числа {12345}: {15}")
+
+
+
 
 # task 7
-"""  Потрібно написати програму, яка просить користувача ввести числа, доки він не введе 0.
-Програма повинна підрахувати суму всіх введених чисел, окрім 0, і вивести її на екран.
-Це повинно працювать як калькулятор, в який ввів цифру - нажав плюс - ввів наступну цифру.
-Після введеня 0 показує результат сумування.
-Розв'язати з використанням циклу while та break
+total_sum = 0
+while True:
+     number = input("Введіть число (або 0 для завершення): ")
+     if number == "0":
+           total_sum += int(number)
+           print(f"Сума всіх введених чисел: {total_sum}")
+         
 """
 
 # task 8
-"""  З використанням циклу for реалізуйте гру "Вгадай число".
-Початок програми написаний, гравець має 5 спроб відгадати випадкове число від 1 до 20,
-яке було згенеровано за допомогою функції randint() модуля random.
-У кожній спробі гравець вводить своє припущення, після чого програма повідомляє, чи
-було припущення занадто великим або занадто малим, чи гравець вгадав число.
-"""
-import random
+""" import random
```

Comment:
```suggestion
import random
```
що за зайві символи  навіщо?